docs/external-ip-and-loadbalancer-ingress-in-shared-gateway-mode.md

```diff
@@ -0,0 +1,45 @@
+# External IP and LoadBalancer Ingress in Shared Gateway mode
+
+OVN Kubernetes in Shared Gateway mode implements both External IPs and LoadBalancer Ingress IPs (`service.Status.LoadBalancer.Ingress`) in the form of OVN load balancers. These OVN load balancers live on all of the Kubernetes nodes and are thus highly available and ready for load sharing. A consequence of this highly available design is that OVN Kubernetes makes it the administrator's responsibility to route traffic to the Kubernetes nodes for both of these VIP types. 
```

Comment:
```
A consequence of this highly available design is that OVN Kubernetes makes it the administrator's responsibility to route traffic to the Kubernetes nodes for both of these VIP types.
```
umm isn't this what Kubernetes also tells? ingress VIPs and external IPs are supposed to be managed by administrators. This sounds like OVN-K is making a limitation.

Reply:
changed to:

OVN Kubernetes implements both External IPs and LoadBalancer Ingress IPs (`service.Status.LoadBalancer.Ingress`) in the form of OVN load balancers. These OVN load balancers live on all of the Kubernetes nodes and are thus highly available and ready for load sharing. It is the administrator's responsibility to route traffic to the Kubernetes nodes for both of these VIP types.